src/circuits/atomic-query-v3-on-chain.ts

```diff
@@ -515,4 +517,15 @@ export class AtomicQueryV3OnChainPubSignals extends BaseConfig {
 
     return this;
   }
+
+  /** {@inheritDoc IGistRootStatePubSignals.getGistRootStatePubSignals} */
+  getGistRootStatePubSignals(): OnChainStateInfo {
```

Comment:
getStatesInfo():StatesInfo 
gist is aslo state

Reply:
renamed